### PR TITLE
Improvement: User/Library header page

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: src/components/avatar/LibraryAvatar.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    .then((icon) => {
+      libraryIcon.value = icon;
+    })
+    .catch(() => {
+      libraryIcon.value = null;
+    });
+}
+
+onMounted(async () => {
+  loadLibraryIcon();
```
Comment (justinehell) [inline src/components/avatar/LibraryAvatar.vue:49]: await loadLibraryIcon();

File: src/components/tooltip/UserTooltip.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+defineProps({
+  user: {
+    type: Object,
+    required: true,
+  },
+  loading: {
+    type: Boolean,
+    default: false,
+  },
```
Comment (justinehell) [inline src/components/tooltip/UserTooltip.vue:47]: I don't see it used in the template

File: src/pages/LibraryPage.vue [hunk 3]
```diff
@@ -88,14 +109,37 @@ import { Notify } from 'quasar';
...
+
+  return UserService.findByLogin(login)
+    .then((data) => {
+      user.value = data;
+    })
+    .catch(() => {
+      user.value = null;
+    }).finally(() => {
+      loadingUser.value = false;
+    });
```
Comment (justinehell) [inline src/pages/LibraryPage.vue:140]: jump line before `.finally(...)`

File: src/pages/LibraryPage.vue [hunk 4]
```diff
@@ -88,14 +109,37 @@ import { Notify } from 'quasar';
...
 const { t } = useI18n();
 const route = useRoute();
 const router = useRouter();
 const library = ref({});
+const user = ref(null);
+const loadingUser = ref(true);
 const currentTab = ref('information');
 
+/**
+ * Load user picture by its login.
```
Comment (justinehell) [inline src/pages/LibraryPage.vue:125]: just `Load user by its login` no?

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/components/avatar/LibraryAvatar.vue (hunk 1 above)
Comment (Zorin95670) [reply src/components/avatar/LibraryAvatar.vue:49]: done

File: src/components/tooltip/UserTooltip.vue (hunk 2 above)
Comment (Zorin95670) [reply src/components/tooltip/UserTooltip.vue:47]: done

File: src/pages/LibraryPage.vue (hunk 4 above)
Comment (Zorin95670) [reply src/pages/LibraryPage.vue:125]: yes done!

**Review by justinehell — APPROVED** [review]

**Pull request merged**